src/main/scala/geotrellis/sdg/PopulationNearRoads.scala

```diff
@@ -78,7 +78,7 @@ object PopulationNearRoads extends CommandApp(
 
         import scala.concurrent.ExecutionContext.Implicits.global
 
-        val result: Map[Country, PopulationSummary] =  {
+        val result: List[(Country, PopulationSummary, Array[Double])] = {
```

Comment:
Fine for now, next step this will need to return histogram as well, this tuple should then become a case class.